builder.go

```diff
@@ -283,6 +296,7 @@ func (k *kResolver) watch() error {
 			if hasMore {
 				k.handle(up.Object)
 			} else {
+				k.lastResolve.Set(float64(time.Since(k.lastResolveTime).Seconds()))
```

Comment:
@lalalalatt do you achieve what do you want with this change or did your test it?  I don't think that lastResolve will update frequently therefore what you will see is "0" on prometheus metrics.